admin-frontend/app_singleapp/lib/widgets/features/table-expanded-view/create-strategy-widget.dart

```diff
@@ -34,15 +35,16 @@ class _CreateValueStrategyWidgetState extends State<CreateValueStrategyWidget> {
   bool isUpdate = false;
   bool isTotalPercentageError = false;
 
+  List<RolloutStrategyAttribute> rolloutStrategyAttributeList = [];
+
   @override
   void initState() {
     super.initState();
     if (widget.rolloutStrategy != null) {
       _strategyName.text = widget.rolloutStrategy.name;
       _strategyPercentage.text = widget.rolloutStrategy.percentageText;
```

Comment:
this initState works because there are never two of these on the screen at the same time.